model/accounts_mgmt/v1/quota_cost_type.model

```diff
@@ -19,5 +19,5 @@ struct QuotaCost {
 	QuotaID string
 	Allowed integer
 	Consumed integer
-	RelatedResources []interface
+	RelatedResources RelatedResources
```

Comment:
This used to be an array and it is now a struct? Can you share an example of the actual data to verify?

Reply:
yeah I think I should change it back to array - my bad.